### ISSUE TITLE
Forum does not like topics without a post

### ISSUE THREAD

**Issue by gunchleoc** [widelands/widelands_website#26]
As soon as the last post in a topic is deleted, this topic can no longer be listed in the admin view or on the site. It results in an internal server error. The quickfix is to delete all topics without posts using manage.py shell and the solution is to check whenever a post is deleted if the containing topic is empty. If it is, it should be deleted as well.

**Comment by gunchleoc** [comment]
(by sirver)
hmpf - I just tried to reproduce this on alpha.wl.org, but I failed. 

There is clearly code that deletes a topic as soon as you delete the last post in it (in pybb/models.py) and it did work as advertised on alpha when I deleted in the forum and also when I deleted on the admin page. 

I am absolutely sure that the bug we saw on the website twice this week comes from topics without any posts and I know that the posts got deleted (through the admin panel), but I have no idea how to repro - so setting to incomplete, maybe it shows its ugly head again.
